src/vs/workbench/contrib/logs/electron-browser/logsActions.ts

```diff
@@ -26,3 +27,22 @@ export class OpenLogsFolderAction extends Action {
 		return this.electronService.showItemInFolder(URI.file(join(this.environmentService.logsPath, 'main.log')).fsPath);
 	}
 }
+
+export class OpenExtensionLogsFolderAction extends Action {
+
+	static readonly ID = 'workbench.action.openExtensionLogsFolder';
+	static readonly LABEL = nls.localize('openExtensionLogsFolder', "Open Extension Logs Folder");
+
+	constructor(id: string, label: string,
+		@IEnvironmentService private readonly environmentService: IEnvironmentService,
+		@IElectronService private readonly electronService: IElectronService,
+		@IElectronEnvironmentService private readonly electronEnvironmentService: IElectronEnvironmentService,
+	) {
+		super(id, label);
+	}
+
+	run(): Promise<void> {
+		const extensionLogsPath = URI.file(join(this.environmentService.logsPath, `exthost${this.electronEnvironmentService.windowId}`, 'exthost.log')).fsPath;
```

Comment:
I guess ideally we would share this path with the extension service, let me know if you have a good suggestion for where to put it. Can't be on `IExtensionService` since it's only relevant to desktop.

Reply:
- You can move extension host logs location to `ElectronEnvironmentService` and can be reused in `ExtensionService`.
- Instead of hard coding `exthost.log`, can you read that folder and reveal the first child? Feel free to do the same for `OpenLogsFolderAction`

Reply:
OpenLogsFolderAction does the same - it hardcodes 'main.log'. I can't figure out how to get Finder to actually open the folder without mentioning a specific item inside it.

Reply:
I dunno if ElectronEnvironmentService feels like the right place for it. Does it make sense to have in the normal EnvironmentService?

Reply:
I think `ElectronEnvironmentService` is the appropriate place as extHostLog is available only for VS Code Desktop Local Ext host which is the exact scope of this service

Reply:
> OpenLogsFolderAction does the same - it hardcodes 'main.log'. I can't figure out how to get Finder to actually open the folder without mentioning a specific item inside it.

True, that's why I suggest to get an entry from the folder and use it to open.

Reply:
Oh, I misunderstood the suggestion, yeah that makes sense.

To me it seems like ElectronEnvironmentService has things relevant to electron itself, not the environment in an electron window. I found `NativeWorkbenchEnvironmentService` which looks relevant to me.

Reply:
Eh nevermind, I don't understand IWorkbenchEnvironment service. Do you know what the logsPath on BrowserWorkbenchEnvironmentService actually means?

![image](https://user-images.githubusercontent.com/323878/74383901-f47c8d00-4da4-11ea-8242-34def0398ede.png)


I'll just put it in ElectronEnvironmentService...